glustercli/cmd/glustershd.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/glustershd/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	helpSelfHealCmd     = "Gluster Self Heal"
+	helpSelfHealInfoCmd = "Self Heal Info"
+)
+
+var (
+	// Heal Info Flags
+	flagSummaryInfo    bool
+	flagSplitBrainInfo bool
+
+	// Volume Name
+	volname string
```

Comment:
This need not be a global variable as it's not used to store value of a flag. You can already access volume name using the variable `volname := args[0]`